### PR TITLE
feat: add blog post card component

### PR DESCRIPTION
**Pull request by seymenklc**
What was done

added arrow upright icon 
created blog post card without labels

What To do; label component will be added

### CODE REVIEW

File: components/Blog/Card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import Description from '@/components/Blog/Card/Description';
+import Img from '@/components/Blog/Card/Img';
+import Info from '@/components/Blog/Card/Info';
+import Title from '@/components/Blog/Card/Title';
+
+interface Props {
+   children: React.ReactNode;
+}
+
+// TODO: Add Card Labels
```
Comment (eraygundogmus) [inline components/Blog/Card/index.tsx:10]: we can remomve that, as @oguzergul said, there will be a docs for components soon

File: components/Blog/Card/Info.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+interface Props {
+   children: React.ReactNode & {
+      [index: number]: string;
+   };
+}
+
+const Info = ({ children }: Props) => {
+   return (
+      <div className="text-sm font-semibold text-indigo-700">
+         <span>{children[0]}</span>
```
Comment (eraygundogmus) [inline components/Blog/Card/Info.tsx:10]: it does not seem like a healty way to do that, is not it kinda over engineering?

File: components/Blog/Card/Title.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+interface Props {
+   link: LinkProps;
+   children: React.ReactNode;
+}
+
+const Title = ({ children, link }: Props) => {
+   return (
+      <div className="flex items-center justify-between">
+         <h3 className="text-2xl font-semibold text-gray-900">{children}</h3>
```
Comment (eraygundogmus) [inline components/Blog/Card/Title.tsx:12]: what happens if I click on this title? is not it better to routing on text too like an anchor tag?

**Review by eraygundogmus — CHANGES_REQUESTED** [review]

File: components/Blog/Card/Title.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+         <Link {...link}>
+            <Image
```
Comment (oguzergul) [inline components/Blog/Card/Title.tsx:14]: You should not use svgs as Image. Svg is not an image type. It is vector. Please use svg component library which already installed to projrct named as react-feather. Or you can add new svg icons and use with svgr.

File: components/Blog/Card/index.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+ *      <Card.Info link={{Link Props}} >{['isim', 'tarih']}</Card.Info>
+ *      <Card.Title>Card Title</Card.Title>
+ *      <Card.Description>
+ *          Lorem ipsum dolor sit amet consectetur adipisicing elit.
+ *     </Card.Description>
+ *     <Card.Labels></Card.Labels>
+ *   </Card>
+ *  */
+
+const Card = ({ children }: Props) => {
```
Comment (oguzergul) [inline components/Blog/Card/index.tsx:26]: This component is specific for Blog. Component name should be BlogCard.

File: components/Blog/Card/Img.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import Image, { ImageProps } from 'next/image';
+
+interface Props extends ImageProps { }
+
+const Img = ({ ...props }: Props) => {
```
Comment (oguzergul) [inline components/Blog/Card/Img.tsx:5]: We dont need to divide BlogCard into seperate components. You can use Compound Component Pattern for createing it more simple way. Create only BlogCard file and implement all componnents into this file. And delete all other files.You can learn it by watching this video or you can inspect codes on this repository. https://youtu.be/4xFf_KiBE6A

**Review by oguzergul — CHANGES_REQUESTED** [review]

**Pull request closed without merging**